README.md

```diff
@@ -2,14 +2,12 @@
 This analysis retrieves the device list of your account and print to the console.
```

Comment:
This analysis retrieves the device list of your account and prints it in the console.